pkg/controller/servicebindingrequest/binder.go

```diff
@@ -465,28 +467,28 @@ func nestedMapComparison(a, b *unstructured.Unstructured, fields ...string) (boo
 // update the list of objects informed as unstructured, looking for "containers" entry. This method
 // loops over each container to inspect "envFrom" and append the intermediary secret, having the same
 // name than original ServiceBindingRequest.
-func (b *Binder) update(objs *unstructured.UnstructuredList) ([]*unstructured.Unstructured, error) {
+func (b *Binder) update(objs []*unstructured.Unstructured) ([]*unstructured.Unstructured, error) {
```

Comment:
Is this change from `*unstructured.UnstructuredList` to `[]*unstructured.Unstructured` essential for this change?